tests/acceptance/features/apiSpaces/search.feature

```diff
@@ -47,8 +47,6 @@ Feature: Search
     Then the HTTP status code should be "207"
     And the search result should contain "4" entries
     And the search result of user "Brian" should contain these entries:
-      | /folder                                           |
-      | /folder/SubFolder1                                |
-      | /folder/SubFolder1/subFOLDER2                     |
-      | /folder/SubFolder1/subFOLDER2/insideTheFolder.txt |
-
+      | /SubFolder1                                |
+      | /SubFolder1/subFOLDER2                     |
+      | /SubFolder1/subFOLDER2/insideTheFolder.txt |
```

Comment:
The search result has 4 entries. But we only check for 3 entries.
What is the 4th entry that is returned?

Reply:
I tried it explain in description https://github.com/owncloud/ocis/pull/4463#issue-1351837237. 
/folder is mountpoint and looks like id in the search result